Shared/Samples/Edit features using feature forms/EditFeaturesUsingFeatureFormsView.swift

```diff
@@ -103,102 +103,60 @@ struct EditFeaturesUsingFeatureFormsView: View {
+                    featureLayer.clearSelection()
+                }
+                .sheet(isPresented: $isShowingFeatureForm) {
                     if let featureForm {
-                        VStack {
-                            featureFormToolbar
-                            
-                            // Displays the feature form using the toolkit component.
-                            FeatureFormView(featureForm: featureForm)
-                                .padding(.horizontal)
-                                .task {
-                                    defer { hasEdits = false }
-                                    
-                                    for await hasEdits in featureForm.$hasEdits {
-                                        self.hasEdits = hasEdits
-                                    }
+                        // Displays the feature form using the toolkit component.
+                        FeatureFormView(root: featureForm, isPresented: $isShowingFeatureForm)
+                            .onFormEditingEvent { event in
+                                switch event {
+                                case .discardedEdits:
+                                    isShowingFeatureForm = false
+                                case .savedEdits:
+                                    isApplyingEdits = true
+                                @unknown default:
+                                    fatalError("Unknown form editing event: \(event)")
                                 }
-                                .task {
-                                    defer { hasValidationErrors = false }
-                                    
-                                    for await validationErrors in featureForm.$validationErrors {
-                                        hasValidationErrors = !validationErrors.isEmpty
-                                    }
+                            }
+                            .task {
+                                defer { hasEdits = false }
+                                
+                                for await hasEdits in featureForm.$hasEdits {
+                                    self.hasEdits = hasEdits
                                 }
-                        }
-                    }
-                }
-                .alert("Discard Edits", isPresented: $isShowingDiscardEditsAlert) {
-                    Button("Cancel", role: .cancel) {}
-                    Button("Discard", role: .destructive) {
-                        featureForm?.discardEdits()
-                        endEditing()
+                            }
+                            .task {
+                                defer { hasValidationErrors = false }
+                                
+                                for await validationErrors in featureForm.$validationErrors {
+                                    hasValidationErrors = !validationErrors.isEmpty
+                                }
```

Comment:
In my opinion, tracking this was more relevant before, but now the component itself also alerts the user to validation errors. I'll let you make the call, but I think it can be removed.

<p align="center">
  <img width="25%" src="https://github.com/user-attachments/assets/e298c565-fd30-4fe5-89e9-3d953f7997fd">
</p>